### ISSUE TITLE
<aside> landmark is nested in <main> landmark on the dashboard page

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#10873]
**Describe the bug**

On the dashboard page, /dashboar, the `<aside>` landmark is nested in a `<main>` landmark on the dashboard page. This is semantically incorrect and can affect screen reader users.

**To Reproduce**

1. Navigate to the dashboard page, https://dev.to/dashboard
1. Open the browser dev tools and search for the `<aside>` opening tag. Notice that it is nested in the pages `<main />` tag.

![image](https://user-images.githubusercontent.com/833231/96189552-afe65a00-0f0e-11eb-926d-174b84657414.png)

**Expected behavior**

The `<aside />` landmark HTML tag should not be nested in the `<main />` landmark HTML tag.

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/96189274-2df63100-0f0e-11eb-9239-530fd9772c3e.png)

**Desktop (please complete the following information):**

- OS, version: N/A
- Browser, version: N/A

**Smartphone (please complete the following information):**

- Device: N/A
- OS, version: N/A
- Browser, version: N/A

**Additional context**

**Comment by clandau** [comment]
I'll take this!

**Comment by nickytonline** [comment]
Awesome. I've assigned it to you @clandau.

**Comment by clandau** [comment]
FYI this will actually be @andreamartz 's pull request, I'm helping her as a mentor. 
Can we get some direction please on how we should handle this? 

I see 2 ways:
To rename the tag to `div` or something more appropriate. 
Or, move the main tag to instead surround just the "posts" section. `<div class="crayons-layout__content flex flex-col">`

Thanks!

**Comment by nickytonline** [comment]
Thanks for the update @clandau. I'll assign it to her instead then.

**Comment by nickytonline** [comment]
I think you suggestion of moving the `<main />` tag sounds good.

i.e. `<div class="crayons-layout__content flex flex-col">...</div>` becomes `<main class="crayons-layout__content flex flex-col">...</main>`

**Comment by nickytonline** [comment]
@colabottles, might have some suggestions here too.

**Comment by colabottles** [comment]
`<aside>` is related to the primary content of the web page but does not constitute the main intent of the primary page. So it should be nested outside of the `<main>...</main>` div.

**Comment by clandau** [comment]
unassigned myself, neither Andrea nor myself will have the time. Sorry!

**Comment by samrae7** [comment]
I would like to take this

**Comment by nickytonline** [comment]
Awesome @samrae7! I've assigned it to you.

**Comment by samrae7** [comment]
Apologies for the delay with this. Today I tried to get started but hit the same issue as this:
https://github.com/forem/forem/issues/9532
when trying to setup on my local machine. I don't want to make this change without running and checking on my local.

Will keep trying to resolve but if anyone has a local setup and wants to take this then feel free

**Comment by nickytonline** [comment]
Hi @samrae7. Sorry to hear you're having issues getting installed on your local with Docker. My coworker @jdoss posted yesterday in #9532, https://github.com/forem/forem/issues/9532#issuecomment-744475460 in regards to issues you're having. Let us know if that helps.

**Comment by zeotuan** [comment]
Seem like this issue hasn't been resolved and hasn't been active for a while. I can take on this issue @nickytonline

**Comment by nickytonline** [comment]
@samrae7, I'm assigning this to @zeotuan as we haven't heard back from you since December 2020. Thanks again for willing to take on an issue and we look forward to you contributing to the code base in the future. 😎

**Comment by samrae7** [comment]
@nickytonline really sorry about that. Thanks for following up
